src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -88,14 +88,14 @@ public PowerFxConfig(Features features)
         /// <summary>
         /// Stopgap until Enum Store is refactored. Do not rely on, this will be removed. 
         /// </summary>
-        internal static PowerFxConfig BuildWithEnumStore(CultureInfo cultureInfo, EnumStoreBuilder enumStoreBuilder)
+        internal static PowerFxConfig BuildWithEnumStore(CultureInfo cultureInfo, EnumStoreBuilder enumStoreBuilder, Features features = Features.None)

```

Comment:
> Features features = Features.None [](http://example.com/codeflow?start=116&length=34)

Just to be on safe side regarding internal breaking changes, 

Adding a new default param is a potentially binary breaking change. 
Instead, 
can you add new overloaded method here?